website/content/en/status/report-2023-04-2023-06/freebsd-foundation.adoc

```diff
@@ -67,35 +67,35 @@ If this is you and we have not yet connected, please schedule a call on link:htt
 Some other partnership-related activities this quarter:
 
 * I created link:https://docs.google.com/presentation/d/1tDCpbfxbqIucmJF6H15vK-ETrQsCMOVtxoqLem_V0Z0/edit?usp=sharing[these slides] about how partnering with the Foundation helps advance FreeBSD.
-  If you have ideas for how I can improve these slides, or would like me to present them to your organization, please link:mailto:greg@freebsdfoundation.org[send me an email], or link:https://calendly.com/greg-freebsdfound/30min[schedule a call].
+  If you have ideas for how I can improve these slides, or would like me to present them to your organization, please mailto:greg@freebsdfoundation.org[send me an email], or link:https://calendly.com/greg-freebsdfound/30min[schedule a call].
   And please feel free to share the presentation liberally, in whole or in part.
 * I worked with my Foundation colleagues to create a number of industry-specific use case slides for a presentation to an industry analyst.
 * I am also pursuing grant opportunities with bodies including:
 ** NSF Secure and Trustworthy Cyberspace (SaTC)
 ** Sovereign Tech Fund
-** NGI
+** NGI.
 
 In terms of research, my broad aim is to make sure that all of the expertise in this community is reflected in the global conversations on computing performance, security, and energy efficiency.
 As a community, we have much to bring to this work.
 
 So far, I have been tracking and plugging into the following threads:
 
 * link:https://openforumeurope.org/open-source/[Open Forum Europe]
-* link:https://www.nist.gov/chips/research-and-development-program[CHIPS Research and Development]
+* link:https://www.nist.gov/chips/research-and-development-program[CHIPS Research and Development].
 
-If you have research ideas or are interested in working together in this area, please link:mailto:greg@freebsdfoundation.org[send me an email], or link:https://calendly.com/greg-freebsdfound/30min[schedule a call].
+If you have research ideas or are interested in working together in this area, please mailto:greg@freebsdfoundation.org[send me an email], or link:https://calendly.com/greg-freebsdfound/30min[schedule a call].
 
 ==== OS Improvements
 
-During the second quarter of 2023, 339 src, 155 ports, and 20 doc tree commits identified the FreeBSD Foundation as a sponsor.
+During the second quarter of 2023, 339 src, 155 ports, and 20 doc tree commits identified The FreeBSD Foundation as a sponsor.
 Some of this and other Foundation-sponsored work is described in separate report entries:
 
 * Continuous Integration
 * FreeBSD as a Tier 1 cloud-init Platform
 * OpenSSL 3 in base
 * OpenStack on FreeBSD
-* Security Sandboxing Using man:ktrace[1]
-* SIMD enhancements for amd64
+* Security Sandboxing Using ktrace(1)
```

Comment:
Can you please explain why you would like to remove the use of the `man` macro?

Reply:
Ah,I forgot. From [`fd6c477` (#221)](https://github.com/freebsd/freebsd-doc/pull/221/commits/fd6c4773da6440a31df862b6588e71c82ebd70da): 

> - prepare for appearance of `_index.adoc` anchors and then link,
  internally, from each title, to the anchor for the relevant section
  within the overall report.

The sentence that precedes the six bullet points: 

> Some of this and other Foundation-sponsored work is described in separate report entries:

– so, each of the six list items can become a link (and in this context, it will be overkill to have a link to the manual page).

Reply:
I like this idea. Can you please add the links within this pull request? If not I will do it as soon as possible.

Reply:
I'm quite tuned-out, but I'll try to give it thought; should be easier for me/someone to get a head around after this PR is closed _and_ the index page is in place. 

IIRC this internal linking was done from a Foundation sub-report in a past sub-report.

Reply:
I have tried to implement your suggestion. If it does not work we can always fix it later.